app/connectors/SubmittedProcessConnector.scala

```diff
@@ -0,0 +1,40 @@
+/*
```

Comment:
Can you run "sbt scalafmt", to reformat the code and push. If you are using intelij, it has way to run it also, Richard or Mac will know

Reply:
I tend to run "sbt scalafmtAll" before pushing to the origin.

Reply:
I ran scalafmt before pushing latest changes